src/img/icons/circle-thumbs-down.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 30 30" fill="none">
+<rect opacity="0.6" x="0.5" y="0.5" width="26" height="26" rx="13" fill="white" stroke="#FE5E7C"/>
+<path d="M12.1674 15.9975V18.4275C12.1674 18.9109 12.3625 19.3744 12.7098 19.7162C13.0571 20.058 13.5281 20.25 14.0193 20.25L16.4885 14.7825V8.10008L9.52539 8.10008C9.22765 8.09677 8.93872 8.19946 8.71185 8.38924C8.48498 8.57902 8.33544 8.8431 8.2908 9.13282L7.43893 14.6003C7.41207 14.7744 7.42401 14.9522 7.47391 15.1214C7.5238 15.2905 7.61047 15.447 7.72791 15.5799C7.84535 15.7128 7.99074 15.819 8.15402 15.8911C8.3173 15.9632 8.49456 15.9995 8.67352 15.9975L12.1674 15.9975ZM16.4885 8.10008H18.3404C18.6678 8.10008 19.2663 8.10008 19.5749 8.10008C19.5749 8.40383 19.575 8.99283 19.575 9.31507V13.5675C19.575 13.8898 19.5749 14.175 19.5749 14.7825C18.9576 14.7825 18.6678 14.7825 18.3404 14.7825H16.4885" fill="#FE5E7C"/>
+<path d="M16.4885 14.7825L14.0193 20.25C13.5281 20.25 13.0571 20.058 12.7098 19.7162C12.3625 19.3744 12.1674 18.9109 12.1674 18.4275V15.9975L8.67352 15.9975C8.49456 15.9995 8.3173 15.9632 8.15402 15.8911C7.99074 15.819 7.84535 15.7128 7.72791 15.5799C7.61047 15.447 7.5238 15.2905 7.47391 15.1214C7.42401 14.9522 7.41207 14.7744 7.43893 14.6003L8.2908 9.13282C8.33544 8.8431 8.48498 8.57902 8.71185 8.38924C8.93872 8.19946 9.22765 8.09677 9.52539 8.10008L16.4885 8.10008M16.4885 14.7825V8.10008M16.4885 14.7825H18.3404C18.6678 14.7825 18.9576 14.7825 19.5749 14.7825C19.5749 14.175 19.575 13.8898 19.575 13.5675V9.31507C19.575 8.99283 19.5749 8.40383 19.5749 8.10008C19.2663 8.10008 18.6678 8.10008 18.3404 8.10008H16.4885" stroke="#F9FAFA" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
These are such bad **single-use** icons ;(

This is not your fault, it's because of the designs, and that they need to have basically three colors in them:
- green
- green w/ opacity
- grey

This breaks SVG so bad, as it prevents us from setting colors programmatically. Plus they are not really optimized. I hate them.

I've optimized the file a bit, so please replace the file with this one:
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg viewBox="0 0 30 30"><path d="M 15,30 C 23.30039,30 30,23.30039 30,15 30,6.69961 23.30039,0 15,0 6.69961,0 0,6.69961 0,15 0,23.30039 6.69961,30 15,30 Z m 0,-1.111111 C 7.2959456,28.888889 1.1111111,22.704054 1.1111111,15 1.1111111,7.2959456 7.2959456,1.1111111 15,1.1111111 c 7.704054,0 13.888889,6.1848345 13.888889,13.8888889 0,7.704054 -6.184835,13.888889 -13.888889,13.888889 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:0.6;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#68d2d2;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /><path d="M 16.480667,12.225 V 9.5249889 C 16.480667,8.9879222 16.263889,8.4728667 15.878,8.0931111 15.492111,7.7133444 14.968778,7.5 14.423,7.5 l -2.743556,6.075 v 7.424889 h 7.736778 c 0.330889,0.0037 0.651889,-0.110445 0.903889,-0.321222 0.252111,-0.210889 0.418333,-0.504334 0.467889,-0.826223 l 0.946556,-6.075 c 0.02978,-0.193444 0.01656,-0.391 -0.03889,-0.579 C 21.640222,13.010556 21.543889,12.836667 21.413444,12.689 21.283,12.541333 21.121444,12.423333 20.94,12.343222 20.758556,12.263111 20.561556,12.222778 20.362778,12.225 Z m -4.801223,8.774889 H 9.6218222 c -0.3638222,0 -1.0287666,0 -1.3717333,0 2.22e-5,-0.337445 -3.33e-5,-0.991889 -3.33e-5,-1.35 V 14.925 c 0,-0.358111 8.88e-5,-0.675 8.88e-5,-1.35 0.6858778,0 1.0078667,0 1.3716778,0 h 2.0576218" style="fill:#68d2d2;stroke-width:1.11111116" /><path d="M 14.423828,6.9453125 A 0.55561114,0.55561114 0 0 0 13.916016,7.2714844 L 11.320312,13.019531 H 9.6210938 8.25 a 0.55561114,0.55561114 0 0 0 -0.5546875,0.554688 c 0,0.674951 0,0.993359 0,1.351562 v 4.72461 c 0,0.358168 2.22e-5,1.012272 0,1.349609 A 0.55561114,0.55561114 0 0 0 8.25,21.554688 h 1.3710938 1.9570312 a 0.55561114,0.55561114 0 0 0 0.199219,0 h 7.632812 c 0.462575,0.0051 0.911225,-0.152793 1.265625,-0.449219 0.354473,-0.296514 0.591848,-0.711622 0.66211,-1.167969 l 0.945312,-6.074219 a 0.55561114,0.55561114 0 0 0 0,-0.002 c 0.0422,-0.274119 0.02384,-0.554034 -0.05469,-0.820312 -0.07855,-0.266185 -0.214572,-0.512563 -0.398438,-0.720704 -0.183904,-0.208185 -0.411637,-0.372063 -0.666016,-0.484374 -0.254375,-0.112313 -0.528707,-0.169092 -0.80664,-0.166016 H 17.037109 V 9.5253906 c 0,-0.6865605 -0.278786,-1.3451805 -0.769531,-1.828125 -0.490727,-0.4829413 -1.153194,-0.7519531 -1.84375,-0.7519531 z m 0.302734,1.2324219 c 0.274885,0.061708 0.560575,0.1125942 0.761719,0.3105468 0.281034,0.2765668 0.4375,0.6495365 0.4375,1.0371094 v 2.6992184 a 0.55561114,0.55561114 0 0 0 0.554688,0.556641 h 3.882812 a 0.55561114,0.55561114 0 0 0 0.0059,0 c 0.119623,-0.0013 0.23719,0.0224 0.345703,0.07031 0.108509,0.04791 0.206219,0.11793 0.283203,0.205079 0.07702,0.08719 0.131722,0.189236 0.164062,0.298828 0.03236,0.109722 0.0408,0.225121 0.02344,0.33789 l -0.947266,6.074219 c -0.02886,0.187431 -0.125642,0.359111 -0.27539,0.484375 -0.1496,0.125127 -0.341812,0.195567 -0.541016,0.193359 a 0.55561114,0.55561114 0 0 0 -0.0059,-0.002 H 12.234375 V 13.695312 Z M 8.8066406,14.130859 h 0.8144532 1.5019532 v 6.3125 H 9.6210938 8.8046875 c -1.13e-5,-0.290616 0,-0.582592 0,-0.792968 v -4.72461 c 0,-0.210569 0.00193,-0.531558 0.00195,-0.794922 z" style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;text-orientation:mixed;dominant-baseline:auto;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;vector-effect:none;fill:#f9fafa;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:1.11111116;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" /></svg>
```

Note that the above changes the size to actually be `30x30` so you might need to adjust your styles